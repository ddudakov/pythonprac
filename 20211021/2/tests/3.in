:sum_mul a b c d (a+b)*(c+d)
sum_mul 1 2 3 4
sum_mul 2 3 1 4
quit "{}:{}"
