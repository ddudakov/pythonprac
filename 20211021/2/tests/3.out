21
25
2:4
